changelog/64286.fixed.md

```diff
@@ -0,0 +1 @@
+Fix 2 pkgrepo.absent bugs for apt-based distros
```

Comment:
Can  you clarify what bugs this is resolving.

Reply:
The reason I left it this way is because I figured the changelog would link to the issue, which would have the further explanation. I can try to summarize this though. I'll update this when I get tests written, hopefully by the end of the week.

Reply:
As an FYI, the issue(s) do get linked on the rendered changelog.
A good summary however goes a long way.

Thank You @terminalmage for the issue and the PR to fix it